x-pack/plugins/security_solution/common/machine_learning/empty_ml_capabilities.ts

```diff
@@ -37,6 +37,7 @@ export const emptyMlCapabilities: MlCapabilitiesResponse = {
     canDeleteDataFrameAnalytics: false,
     canCreateDataFrameAnalytics: false,
     canStartStopDataFrameAnalytics: false,
+    canCreateMlAlerts: false,
```

Comment:
Just to clarify, is there any expectation we check this permission and restrict access within Detections for creating ML Rules? I imagine no since the Detection Engine is managed as a separate application, and this seems specific to the new ML Alert Type, but want to make sure we're respecting permissions as expected from the ML permissions model perspective.

i.e Should we disable the ML Detections Rule type if `canCreateMlAlerts:false`?
<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/107971705-096b9d80-6f70-11eb-8f95-e23356248f54.png" />
</p>

Reply:
hey @spong! 👋 I reckon `canCreateMlAlerts` should not affect the ML Detections Rule in Security app, as it's dedicated to the ML related alert types only. No actions are required on your side